infra/staging/main.tf

```diff
@@ -0,0 +1,137 @@
+###############
+## Variables ##
+###############
+
+variable "project_id" {
+  type = string
+  default = "call-home-staging"
+}
+
+variable "location" {
+  type = string
+  default = "asia-southeast1"
+}
+
+variable "app_bucket_name" {
+  type = string
+  default = "call-home-staging-strapi-app-bucket"
+}
+
+variable "db_name" {
+  type = string
+  default = "strapi"
+}
+
+variable "db_instance_name" {
+  type = string
+  default = "strapi-db-instance"
+}
+
+variable "db_username" {
+  type = string
+  sensitive = true
+}
+
+variable "db_password" {
+  type = string
+  sensitive = true
+}
+
+######################
+## Project Settings ##
+######################
+
+data "google_project" "project" {
+  project_id = var.project_id
+}
+
+// Enable Cloud SQL Admin API to connect to database
```

Comment:
To be honest I'm not sure, in fact I'm not even sure if we need to enable this cloud SQL API at all? For some reason I think I needed it when doing the experiments, but I just tried terraform applying the DB infra with this API disabled and that worked well. Maybe it's needed for the GAE app to connect to the DB then? I can't be sure though until we actually deploy the app, so I've commented it out and left a TODO there - would this work for now?

Reply:
I think it's probably better to just delete the lines instead of commenting them out. These sorts of "maybe we need this idk" comments have a tendency to get stale, especially if

a) we don't consciously look at them (maybe doesn't apply, since you are looking at this actively)
b) we don't leave clues as to when we should have already deleted them (e.g. "We can delete this if Strapi runs fine in staging without it."), and by the time we see them again we've forgotten the original context.

In general, I prefer to have these sorts of comments in a more ephemeral medium like the PR message, design doc or ticket. But it's not a big enough deal to block submission.

Reply:
Have removed the commented-out cloud SQL API in https://github.com/bettersg/call-home/pull/87/commits/a5d163e756a4429ef771f8ce2174d2249c6f2714